### DISCUSSION TITLE
What does the trafficsplit_subscribers and trafficsplit_updates metrics measure?

### DISCUSSION THREAD

**Discussion by gabrielmcf** [linkerd/linkerd2#4931] [General] [answered]
I wasn't able to find an answer neither in linkerd2 documentation or google. I only found two issues with an prometheus return example containing them. I've seen them in prometheus while I was looking for a way to build a Grafana dashboard to look into traffic splits.

**Comment by ihcsim** [comment, marked as answer]
They are metrics exposed by the `linkerd-destination` service, showing the number of traffic split subscribers and updates to a traffic split resource. Depending on what you hope to get from your Grafana dashboard, they are probably implementation details that you don't need to worry about. The traffic split page on the Linkerd dashboard has metrics that should be relevant to your dashboard.

**Reply by gabrielmcf** [reply]
The subscribers are the endpoints of the service? Or the clients doing requests?

**Reply by ihcsim** [reply]
They subscribe to the K8s API endpoint and watch for changes to the `TrafficSplit` resources.
